_posts/2022-09-12-changelog-178.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #178"
+date: 2022-09-12 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9268], [#9259] Complete proc macros in attributes
+
+  {% include gif-img.html path="/assets/posts/changelog-178/complete_attr_proc_macros" w="350px" %}
+
+* [#9251] Detect [E0728](https://doc.rust-lang.org/error-index.html#E0728) compiler error
+
+  {% include gif-img.html path="/assets/posts/changelog-178/await_outside_async" w="350px" %}
+
+* [#9272] Detect [E0434](https://doc.rust-lang.org/error-index.html#E0434) compiler error
+
+  {% include gif-img.html path="/assets/posts/changelog-178/use_outer_variable" w="350px" %}
+
+* [#9209] Provide debugger pretty-printers to render range types (e.g. `core::ops::range::Range`)
+
+  <img src="/assets/posts/changelog-178/pretty_printers_range_types.png" width="700px"/>
+
+* [#8961] Take into account build target during collection information about the project model and don't download and analyze [platform-specific](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies) dependencies not suitable for the current platform
+
+* [#9256] Detect [E0733](https://doc.rust-lang.org/error-index.html#E0733) compiler error
```

Comment:
Let's mention that we still don't detect complex cases like described in https://github.com/intellij-rust/intellij-rust/issues/9319